index.html

```diff
@@ -0,0 +1,163 @@
+            </p>
+        </div>
+        <div class="ignite">
+            <img src="asset/CC ignite icon.png">
+            <p class="p-list">CC Ignite</p>
+            <p class="p-det">get opportunities to network with CC affiliates around the world, also after the summit.
+            </p>
+        </div>
+        </div>
+        <div class="join-glo">
+        <h2 class="join">Join CC Global Summit 2015</h2>
+    </div>
+    <p class="see-pro">SEE THE HOLE PROGRAM</p>
+    </section>
+
+    <div class="f-speakers">
+        <h1>Featured Speakers</h1>
+        <hr>
+    </div>
+    <section class="featured-speakers" id="featured-section">
+    </section>
+    <section class="about-section" id="abt-sec">
+        <div class="card-section">
+        <div class="abt-top">
+        <h1 class="abt-greeting">"Hello! Sharing world"</h1>
+        <h2 class='abt-title'>CREATIVE COMMONS GLOBAL SUMMIT 2015</h2>
+    </div>
+    <div class="abt-det">
+        <p class="abt-detail">The Global Summit brings together the community of experts,academics,and activitsts who comprise the Creative Commons affiliate network in a different country every two years. This year, we hope to expand our invitation list including organizations and individuals.</p>
+    </div>
+        <div class="contact">
+        <p class="abt-contact">Please contact us per Email for nay further questions about CC Global Summit 2015</p>
+        <p class="abt-email">ccsummit2015@cckorea.org</p>
+    </div>
+</div>
+<div class="summit">
+        <h3 class="su-cre">Creative Commons Global Summit 2015 Logo</h3>
+        <hr>
+    </div>
+        <p class="logo">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.</p>
+        <img class="cc-log" src="asset/cc global logo.png" alt="#">
+        <div class="p-pic">
+        <img   class="pic-1" src="asset/1st pc.png">
+        <img class="pic-2" src="asset/pic 2.png">
+    </div>
+
+    </section>
+    <footer class="mobile-footer">
+        <div class="foot">
+        <div class="part">
+        <h2>Partner</h2>
+        <hr>
+    </div>
+        <div class="partners">
+            <ul class="partners-list">
+                <li><img class="mozilla" src="asset/mozilla.png" alt="mozila">
+                </li>
+                <li><img class="google" src="asset/google.png" alt="google">
+                </li>
+                <li><img class="naver" src="asset/naver.png" alt="naver">
+                </li>
+                <li><img class="daumk" src="asset/daumkakaologo.png" alt="daumkakao">
+                </li>
+                <li><img class="air" src="asset/airbnb.png" alt="air">
+                </li>
+
+            </ul>
+        </div>
+        <div class="right-res">
+        <img src="asset/right reserved.svg">
+        <p>2015 Creative Commons Korea.Some Rights Reserved.</p>
+    </div>
+    </footer>
+   
+    <script src="script.js"></script>       
+</body>
```

Comment:
Good job trying to work on the project requirements . I notice You have mixed the home page and about page and made them one page which is not how you were supposed to do the project, you were supposed to create two separate pages namely Home page and about page that are linked together.

Please follow the following requirements when doing this project:
Page Structure and Navigation

The project contains a Home page and About page.
It's possible to navigate between the pages, there's a link from Home page to About page and the opposite too.
Both pages contain the same navbar and footer.


Navbar

Mobile Size

It contains an hamburger menu on the left side.
It's background is transparent or the same color/pattern as the top section background.


Desktop Size

It is divided in a top dark bar and a bottom white bar.
The top short bar section with social media links, logout, and language links aligned to the right.
The taller bottom bar with a logo and a list of page links.
The top bar contains a logo to the left.
The top bar contains a list of page links aligned to the right.


Footer

It contains a logo aligned to the left and text aligned to the right.


Mobile Size

It has a white background.


Desktop Size

About Page: It has a dark background.
Home Page: It has a white background.


Home Page

It contains a top section with a column that includes a small line of text, the page title, a block of text, a medium line of text in bold letters, and a smaller line of text.
It contains a middle section with a column that includes a section header, a divider, five individual articles, each with an image, title, and description, and a line of text at the bottom.
It contains a bottom section with a column that includes a section header, a divider, and six individual articles, each with an image, title, subtitle, and description. Each article has two columns, the first with the image at the left and the second with the title, subtitle, and description.


Mobile Size

The middle section article list is organized in a vertical column.
The middle section individual articles have their elements organized in an horizontal row, with the image at the left, title in the center, and description at the right.
The bottom section articles are organized in a vertical column.


Desktop Size

The middle section article list is organized in an horizontal row.
The middle section individual articles have their elements organized in an vertical column, with the image at the top, title in the center, and description at the bottom.
The bottom section articles are organized in two columns of three articles each.


About Page

It contains a top section with a centered column that includes a page title, a text block with the page description, and a line with a contact information or any other prompt under it.
It contains a middle section with a centered column that includes a section header, a divider, a paragraph of text, and an image or large logo.
It contains a bottom section with a centered column that includes a section header, a divider, a line of text, and two images with text centered over them.


Mobile Size

The bottom section images are organized in a vertical column.


Desktop Size

The bottom section images are organized in an horizontal row.


Design and Styling

The pages should look almost identical to the original design, outside the content differences.
The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.


Basic color swatch: #272A31, #EC5242, #D3D3D3


The pages use the same fonts or simialr fonts from the original design.


Fonts used originally: Cocogoose, Lato


All page elements have similar spacing, borders, shadows, and paddings than the orignal design.


HTML code

The HTML uses semantic tags, sections use section, articles use article, the nav and footer use nav and footer.
There's at least 1, but no more than 1, h1 tag in each page.
The HTML markup is well structured and follows a clear hierarchy, with sections, and articles, each containing appropriated Hx headers and paragraphs.
The page is accessible, images contain alt attributes and there's not images or elements blocking the text in both screen sizes.
The logo contains a hyperlink to the homepage.


CSS code

Selector classes are used multiple times and rules are not repeated, the code is DRY.
The CSS styling is built as mobile-first, media queries are used to change the design from mobile to desktop, not the other way around.
Media queries are properly organized at the bottom of the stylesheet.
NOTE: Students can use CSS frameworks (Bootstrap, Tailwind CSS and any other CSS framework) for styling (if they want to).


Note: It is fine if the student has used Bootstrap or any other framework.


JavaScript code

Dynamically generated HTML

The section "Featured speakers" is generated dynamically with JavaScript.
The JavaScript code is within it's own .js file, not inside the HTML in scripttags.
The list of speakers used to generate the HTML is assigned to a variable as an array or object.
The list of speakers contain an speaker object for each that holds their specific data.


// You can use an array to store the objects
const speakerList = [ { name: "Speaker Name",... } ]
// It's ok to use an object too
const speakerList = { speaker1: { name: "Speaker Name",... } }



The object contains all the information needed to generate the dynamic element, at least the name, role, description, and image url about the speaker. For example:


{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}



There is a function that iterates through the speakers list on window load and creates HTML elements with the data from each object. For example:


window.onload = () => {
  speakerList.map(speaker => {
    // code to build the new HTML element, it can use `createElement()` or just template strings(``).
    // it could attach the new element to a parent inside within the loop
    // or accumulate the generate elements in a variable and attach them all after it
  })
}




The dynamic content is attached to the static HTML by selecting the target parent element and adding the generated HTML with the DOM's appendChild() or innerHTML(). For example:


  const speakerContainer = document.getElementById("element-id")
  ...
  speakerContainer.appendChild(dynamicElements)
  // OR
  speakerContainer.innerHTML = dynamicElements
  // If used inside a loop the innerHTML method should use `+=` instead of just `=`




These are Both your desktop design and mobile design:

![Student html project ](https://github.com/Surafels/Capstone_Project/assets/79506105/9233d5fe-2898-41ec-a775-5c6019dc1084)
![studentmobile](https://github.com/Surafels/Capstone_Project/assets/79506105/b131869e-ce69-482b-80d4-3b11f6aa4870)


Please follow this template design:

Your Home page desktop view should look like this:

![Home Page](https://github.com/Surafels/Capstone_Project/assets/79506105/d6d9eff5-84c9-4841-9d4d-3b7a6f41caad)

Your Home mobile view should look like this:

![Homepage mobile](https://github.com/Surafels/Capstone_Project/assets/79506105/46eb0d92-1d77-4b40-9545-2fe68bf40f40)

Your About page should look like this:

Desktop view:

![About page](https://github.com/Surafels/Capstone_Project/assets/79506105/42362ff7-9b20-4e55-82b6-b17b95951c51)

About Mobile view:

![About page mobile](https://github.com/Surafels/Capstone_Project/assets/79506105/5b37d2c5-df1f-48ae-8c44-70c1ba2061bd)

It is very important to follow project requirements. Please always pay attention to detail.